.ci/templates/toolchain.yml

```diff
@@ -25,7 +25,7 @@ parameters:
 
   - name: VERSION
     type: string
-    default: 'master'
+    default: 'trunk'
```

Comment:
Does this not need changes in scripts to ensure that the layout is properly selected?